code/src/App.js

```diff
@@ -1,9 +1,145 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { QuestionSection } from './QuestionSection'
 
 export const App = () => {
+
+  const groups = ['0-100', '100-200', '500+']
+  const [grams, setGrams] = useState('')
+  const [sugaraddict, setSugaraddict] = useState(false)
+  const [favoriteCandy, setFavoriteCandy] = useState('')
+  const [name, setName] = useState('')
+  const [showSummary, setShowSummary] = useState(false)
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    setShowSummary(true)
+  }
+
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="App">
+
+      <header>
+        <h2>
+          WICH TYPE OF
+        </h2>
+        <h1>
+          CANDY
+        </h1>
+        <h2>
+          ARE YOU?
+        </h2>
+      </header>
+
+      {!showSummary && (
+        <form onSubmit={handleSubmit}>
+
+          {/* HERE YOU CAN CHOOSE A GROUP WITH RADIOBUTTONS */}
+          <QuestionSection>
+            <h3>
+              HOW MANY GRAMS OF LÖSGODIS DO YOU CONSUME PER WEEK?
+            </h3>
+            <div className="gramsWrapper">
+              {groups.map(group => (
+                <label key={group}>
+                  <input
+                    type="radio"
+                    value={group}
+                    onChange={event => setGrams(event.target.value)}
+                    checked={grams === group}
+                  />
+                  {group} g.
+                </label>
+              ))}
+            </div>
+          </QuestionSection>
+
+          {/* HERE YOU FILL IN IF YOURE A SUGARADDICT WITH A CHECKBOX */}
+          <QuestionSection>
+            <h3>
+              ARE YOU ADDICTED TO SUGAR?
+            </h3>
+            <label className="inputWrapper">
+              <div>
+                YES
+                <input
+                  type="checkbox"
+                  checked={sugaraddict}
+                  onChange={event => setSugaraddict(event.target.checked)}
+                />
+              </div>
+            </label>
+          </QuestionSection>
+
+          {/* SELECT YOUR FAVORITE CANDY IN A DROPDOWN MENU */}
+          <QuestionSection>
+            <label className="inputWrapper">
+              <h3>
+                WHAT DO YOU PREFER?
+              </h3>
+              <select
+                onChange={event => setFavoriteCandy(event.target.value)}
+                value={favoriteCandy}
+              >
+                <option value="">Select your favorite candy:</option>
```

Comment:
I guess you don't want this to be an option
```suggestion
                <option value="" disabled>Select your favorite candy:</option>
```